app/services/direction_service.rb

```diff
@@ -0,0 +1,11 @@
+class DirectionService < BaseService
+
+  def self.navigate(start, finish)
+    response = conn('http://www.mapquestapi.com').get("/directions/v2/route") do |f|
+      f.params['key'] = ENV['map_key']
+      f.params['from'] = start
+      f.params['to'] = finish
+    end
+    format_json(response)
+  end
```

Comment:
Great formatting here and also in your other service. I like that you are adding the params/values by setting them on each line.